packages/webpack-extraction-plugin/virtual-loader/index.js

```diff
@@ -0,0 +1,14 @@
+const { URLSearchParams } = require('url');
```

Comment:
More or less the same thing as in:
- https://github.com/atlassian-labs/compiled/blob/master/packages/webpack-loader/src/css-loader.ts
- https://github.com/seek-oss/vanilla-extract/blob/master/packages/webpack-plugin/src/virtualFileLoader.ts

https://webpack.js.org/api/loaders/#inline-matchresource